README.md

```diff
@@ -111,52 +188,93 @@ root error
 
 'eris' also provides developers a way to define a custom format to print the errors. The [`Format`](https://godoc.org/github.com/rotisserie/eris#Format) object defines separators for various components of the error/trace and can be passed to utility methods for printing string and JSON formats.
 
-## Error object
+### Writing your own custom format
 
 The [`UnpackedError`](https://godoc.org/github.com/rotisserie/eris#UnpackedError) object provides a convenient and developer friendly way to store and access existing error traces. The `ErrChain` and `ErrRoot` fields correspond to `wrapError` and `rootError` types, respectively. If any other error type is unpacked, it will appear in the ExternalErr field.
 
 The [`Unpack()`](https://godoc.org/github.com/rotisserie/eris#Unpack) method returns the corresponding `UnpackedError` object for a given error. This object can also be converted to string and JSON for logging and printing error traces. This can be done by using the methods [`ToString()`](https://godoc.org/github.com/rotisserie/eris#UnpackedError.ToString) and [`ToJSON()`](https://godoc.org/github.com/rotisserie/eris#UnpackedError.ToJSON). Note the `ToJSON()` method returns a `map[string]interface{}` type which can be marshalled to JSON using the `encoding/json` package.
 
-## Logging errors with more control
+## Comparison to other packages (e.g. pkg/errors)
 
-While `eris` supports logging errors with Go's `fmt` package, it's often advantageous to use the provided string and JSON formatters instead. These methods provide much more control over the error output and should work seamlessly with whatever logging package you choose. The example below shows how to integrate `eris` with [logrus](https://github.com/sirupsen/logrus).
-
-```golang
-var fields log.Fields
-unpackedErr := eris.Unpack(err)
-fields["method"] = "api.GetResource"
-fields["error"] = unpackedErr.ToJSON(eris.NewDefaultFormat(true))
-logger.WithFields(fields).Errorf("method completed with error (%v)", err)
-```
+### Error formatting and stack traces
 
-When using a JSON logger, the output should look something like this:
+Readability is a major design requirement for `eris`. Similar to the error shown above, this error output shows a wrapped error formatted to easy to interpret JSON.
 
 ```json
```

Comment:
we dont need this json output again here imo. we already showed it twice above. String one makes sense as we use that for comparison

Reply:
sounds good